src/sbd.h

```diff
@@ -62,6 +62,9 @@
 /* exit status for pcmk-servant */
 #define EXIT_PCMK_SERVANT_GRACEFUL_SHUTDOWN 30
 
+/* exit status for cluster-servant */
+#define EXIT_CLUSTER_CMAP_DISCONNECT 40
```

Comment:
Would rather have thought about handling all kinds of disconnect-reconnect this way (EXIT_CLUSTER_DISCONNECT then rather)

Reply:
Renamed. IMO changing other places to follow this pattern is something for another PR.

Reply:
Cleanest thing is probably adding the disconnect abort-stuff and everything around for the cluster-disconnect and rebase this PR on that.

Reply:
Do you mean that the current logic in `sbd_membership_destroy()` should be replace with exit and the situation should always be handled by the inquisitor? What about the cases where the connection loss is very brief and `sbd-cluster` should be able to correct it without restart? This is currently implemented and if we replace it with exit it would be a kind of step back, wouldn't it?

Reply:
That was the idea behind it so we can have quite a simple logic.
If we are fast enough with cmap disconnect we should be with any other disconnect.
Downside is of course that we don't have RR-scheduling & hogged stack for that initial phase.
Which brings me to the initial idea of making any kind of disconnection fatal for the node.
If it manages to reconnect quick enough it is allowed to survive.
On the other hand I'm just realizing a basic difference here between cluster & pacemaker-watcher:
When pacemaker-watcher exits gracefully we can be sure that no resources are running and we thus can disable requiring the strict heartbeat (what I'm doing with the new implementation).